### PR TITLE
Polish a couple things

### PR DESCRIPTION
**Pull request by ryan-pratt**

### CODE REVIEW

File: openc3-cosmos-init/plugins/packages/openc3-vue-common/src/plugins/notify/Toast.vue [hunk 1]
```diff
@@ -73,6 +73,7 @@ export default {
         message: 'Overall message here',
       },
       timeout: null,
+      noToastPaths: ['/login'],
```
Comment (jmthomas) [inline openc3-cosmos-init/plugins/packages/openc3-vue-common/src/plugins/notify/Toast.vue:76]: Good fix, that was bugging me on login

**Review by jmthomas — APPROVED** [review]
I approve but let's have @ryanmelt review

**Comment by jmthomas** [comment]
Here's how the screen overlay works: 
Screenshot 2024-12-13 at 6 22 16 AM

**Comment by jmthomas** [comment]
I fixed the flaky Bucket Explorer playwright test in https://github.com/OpenC3/cosmos/pull/1761/commits/22c4bb016a38789f4bfdc03589a8cb615fc7dbe9

**Review by ryanmelt — APPROVED** [review]

**Pull request merged**